src/main/shaders/standardVertex.vert

```diff
@@ -18,35 +16,17 @@ out vec3 Color;
 const vec3 ka = 0.3*vec3(.5, 0.5, 0.5);
 const vec3 kd = 0.7*vec3(.5, 0.5, 0.5);
 
-// Calculates the dot product d * n, finds the angle between the two, sends
-// angle ratio to lighting luminance.
-// float invRatio(vec3 d, vec3 n) {
-//     // Calculate the cross product between the two vectors
-//     float i, D, N, preAngle;
-//     i = (d[0]*n[0] + d[1]*n[1] + d[2]*n[2]);
-//     D = sqrt(d[0]*d[0]+d[1]*d[1]+d[2]*d[2]);
-//     N = sqrt(n[0]*n[0]+n[1]*n[1]+n[2]*n[2]);
-//     return 1 - rollOff * acos(i/(D*N)) / 3.1415926f;
-
-// }
-
 void main() {
   vec3 lightPosition = directionalLight;
 
-  mat4 Model = move*scale*rotate;
-  vec4 Normal = normalize(Model * vec4(normals, 0.0));
+  vec4 normal = normalize(model * vec4(normals, 0.0));
   const vec3 LightIntensity = vec3(20);
 
   float distance = length(lightPosition - vertexPosition_modelspace);
-  float intensity = dot(Normal, normalize(vec4(lightPosition, 0.0) - vec4(vertexPosition_modelspace, 1.0)));
-  gl_Position = VP * move * scale * rotate *  vec4(vertexPosition_modelspace, 1);
+  float intensity = dot(normal, normalize(vec4(lightPosition, 0.0) - vec4(vertexPosition_modelspace, 1.0)));
+  gl_Position = VP * model * vec4(vertexPosition_modelspace, 1);
```

Comment:
Removing unnecessary math from the shader. This is already calculated once on the CPU side, so just sending the model.